infra/src/main/kotlin/fi/fta/geoviite/infra/inframodel/InfraModelController.kt

```diff
@@ -107,6 +109,13 @@ class InfraModelController @Autowired constructor(
         return pvDocumentService.getDocumentCounts()
     }
 
+    @PreAuthorize(AUTH_ALL_READ)
+    @GetMapping("/velho-import/redirect/{oid}")
```

Comment:
Tämä on nyt urlissa infra-model/velho-import/... ,mikä tuntuu vähän väärältä koska tässä ei ole enää kyse mistään importista eikä myöskään inframodeleista. Toisaalta en oikein tiedä viitsisikö moista varten tehdä omaakaan ja olemassaoleviin ei oikein istu :shrug:

Reply:
Itse ajattelin alunperin että tämä koko velhojuttujen access URL olisi ollut `velho-import`:in sijasta `projektivelho`. Siihen malliin tämä sopisi paremmin

Reply:
Totta. Siinä oli puolestaan se inhottavuus että inframallien edit ja upload ja import ovat kovin samanlaisia (ja samoilla riippuvuuksilla) ja nuo 2 muuta ovat täällä. Suo siellä vetelä täällä. Pitääpä miettiä olisiko tuohon joku fiksumpi.